src/extension/wps/wps-core/src/main/java/org/geoserver/wps/ppio/BinaryPPIO.java

```diff
@@ -14,6 +14,11 @@
  */
 public abstract class BinaryPPIO extends ComplexPPIO {
 
+    protected static final String TILE_WIDTH_KEY = "tilewidth";
```

Comment:
BinaryPPIO has no changes related to the parameters, I'd move these out. In WCS they are specific to GeoTiff, if other formats have their own params they could be different. Suggestion: can we put them into GeoTiffPPIO for the time being, and then move them up as needed if more formats actually start needing them?

Reply:
Well. at the very beginning I have put that on GeoTIFF PPIO indeed. Then I have moved up to BinaryPPIO thinking about more formats starting supporting them in the future. However, we can move up them afterwards, so I'm going to put them back down on GeoTIFF only.